tools/export/ds5_5.py

```diff
@@ -40,6 +40,8 @@ class DS5_5(Exporter):
         's_sources':'2'
     }
 
+    TOOLCHAIN = "ARM"
```

Comment:
The line below (46) seems to tell a slightly different story. Is it possible to replace the access to `TOOLCHAIN` to use `get_toolchain` instead?